Docs: Add note about different license expiry dates (#4517)
elastic/cloud-on-k8s: docs/operating-eck/licensing.asciidoc

== Update your license
Before your current Enterprise license expires, you will receive a new Enterprise license from Elastic (provided that your subscription is valid).

> NOTE: The expiry date of your license can be seen in the license file you received from Elastic. Enterprise licenses are container licenses that are comprised of multiple licenses for individual Elasticsearch clusters with shorter expiry. You will therefore see a different expiry in Kibana or via the Elasticsearch link:https://www.elastic.co/guide/en/elasticsearch/reference/current/get-license.html[`_license`] API. ECK will automatically update the Elasticsearch cluster licenses until the expiry date of the ECK Enterprise license has been reached.

Reviewer: ```suggestion
NOTE: You can see the expiry date of your license in the license file that you received from Elastic. Enterprise licenses are container licenses that include multiple licenses for individual Elasticsearch clusters with shorter expiry. Therefore, you will see a different expiry in Kibana or via the Elasticsearch link:https://www.elastic.co/guide/en/elasticsearch/reference/current/get-license.html[`_license`] API. ECK automatically updates the Elasticsearch cluster licenses as soon as the expiry date of the ECK Enterprise license is reached.
```
